assets.md

```diff
@@ -58,7 +58,7 @@ Link: https://groups.google.com/g/opentelemetry-calendar-contributors
 - Owners: [@open-telemetry/governance-committee](https://github.com/orgs/open-telemetry/teams/governance-committee)
 - Members: SIG maintainers and individuals appointed by maintainers
 
-### OpenTelemetry Calendar Invites Google Group](https://groups.google.com/g/opentelemetry-calendar)
+### [OpenTelemetry Calendar Invites Google Group](https://groups.google.com/g/opentelemetry-calendar)
```

Comment:
personally I don't like section headers to be external links, would rather have that link in the paragraph.

Reply:
The link is actually already listed four lines further down already, let's remove this duplicate from the heading then.

Reply:
```suggestion
### OpenTelemetry Calendar Invites Google Group
```